.claude/skills/website-maintainer/SKILL.md

```
@@ -0,0 +1,40 @@
+---
+name: website-maintainer
+description: Use this skill when working on the Repomix documentation website in `website/` directory, including VitePress configuration, multi-language content, or translation workflows.
+---
+
+# Website Maintainer
+
+VitePress documentation site with 12 languages.
+
+## Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)